README.rst

```diff
@@ -19,36 +19,34 @@
 .. |Binder| image:: https://mybinder.org/badge_logo.svg
    :target: https://mybinder.org/v2/gh/sunpy/sunpy/main?filepath=examples
 
-SunPy is an open-source Python library for Solar Physics data analysis and visualization.
-Our homepage `SunPy`_ has more information about the project.
+``sunpy`` is an open-source Python library for Solar Physics data analysis and visualization.
+`Our homepage has more information about the SunPy project. <https://sunpy.org>`_
 
-For some examples of using SunPy see our `gallery`_, to see the latest changes in SunPy see our `Changelog`_.
+For some examples of using ``sunpy`` see our `gallery`_, to see the latest changes in ``sunpy`` see our `Changelog`_.
 
-.. _SunPy: https://sunpy.org
 .. _gallery: https://docs.sunpy.org/en/stable/generated/gallery/index.html
 .. _Changelog: https://docs.sunpy.org/en/stable/whatsnew/changelog.html
 
 Installation
 ============
 
-The recommended way to install SunPy is with `miniconda`_.
-To install SunPy once conda is installed run the following two commands:
+The recommended way to install ``sunpy`` is with `miniforge`_.
+To install ``sunpy`` once conda is installed run the following command:
 
 .. code:: bash
 
-    $ conda config --append channels conda-forge
     $ conda install sunpy
 
-For detailed installation instructions, see the `installation guide`_ in the SunPy docs.
+For detailed installation instructions, see the `installation guide`_ in the ``sunpy`` docs.
 
-.. _miniconda: https://docs.conda.io/en/latest/miniconda.html
+.. _miniforge: https://github.com/conda-forge/miniforge#miniforge3
```

Comment:
We could do mamba?

Reply:
I'm liking mamba, but we might want to wait until they hit version 1.  Especially for our README.

Reply:
Fair.